src/middlewared/middlewared/test/integration/utils/mock_binary.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding=utf-8 -*-
+import base64
+import contextlib
+import json
+import textwrap
+
+from .call import call
+from .ssh import ssh
+
+
+class BinaryMock:
+    def _load(self):
+        try:
+            return json.loads(ssh("cat /tmp/mocked_binary_launch", check=False).strip())
```

Comment:
Should we statically define the `/tmp/mocked_binary_launch` path since you're using it in multiple places?